operate/client/src/App/DecisionInstance/Header/index.tsx

```diff
@@ -6,18 +6,16 @@
  * except in compliance with the Camunda License 1.0.
  */
 
-import {observer} from 'mobx-react';
```

Comment:
We can remove the `observer` HOC when a component no longer reads any MobX state, right? :)